msu/msu_mod/msu_registry.nut

```diff
@@ -0,0 +1,3 @@
+::MSU.Mod.Registry.addModSource(::MSU.System.Registry.URLDomain.GitHub, "https://github.com/MSUTeam/MSU");
```

Comment:
As previously requested, please move this (adding MSU links) out to a separate PR as this PR should be solely the framework itself.

Reply:
I've fixed this part.